### PR TITLE
[JBEAP-19561] Update shared-resources referenced from EAP QS readmes

### PR DESCRIPTION
**Pull request by emmartins**
Issue: https://issues.redhat.com/browse/JBEAP-19561

### CODE REVIEW

File: guides/USE_JBDS.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ This guide includes the following information.
 * For information about how to import the quickstarts into {JBDSProductName}, and for examples of how to build and deploy a basic quickstart, see link:{LinkGettingStartedGuide}#using_the_quickstart_examples[Using the Quickstart Examples] in the _{GettingStartedBookName}_ for {ProductShortName}.
 * For instructions to run the Arquillian tests using the Maven command line, see link:RUN_ARQUILLIAN_TESTS.adoc#run_the_arquillian_tests[Run the Arquillian Tests].
 
-See https://developers.redhat.com/products/devstudio/overview/[{JBDSProductName}] on the Red Hat Developer Program web site for additional information.
+See https://developers.redhat.com/products/codeready-studio/overview[{JBDSProductName}] on the Red Hat Developer Program web site for additional information.
```
Comment (ncbaratta) [inline guides/USE_JBDS.adoc:47]:
```suggestion
See https://developers.redhat.com/products/codeready-studio/overview[{JBDSProductName}] on the Red Hat Developer Program website for additional information.
```

File: guides/USE_JBDS.md [hunk 2]
```diff
@@ -17,23 +17,23 @@ This guide includes the following information.
...
+For information about how to deploy and run the quickstarts and to run the Arquillian tests in Red Hat CodeReady Studio, see [Run the Quickstarts in Red Hat CodeReady Studio](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/getting_started_guide/index#using_the_quickstart_examples) in the _Getting Started Guide_ for JBoss EAP.
 
 For instructions to run the Arquillian tests using the Maven command line, see [Run the Arquillian Tests](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/RUN_ARQUILLIAN_TESTS.md#run-the-arquillian-tests).
 
-See [Red Hat JBoss Developer Studio](https://developers.redhat.com/products/devstudio/overview/ "Get Started") on the Red Hat Developer Program web site for additional information.
+See [Red Hat Red Hat CodeReady Studio](https://developers.redhat.com/products/codeready-studio/overview/ "Get Started") on the Red Hat Developer Program web site for additional information.
```
Comment (ncbaratta) [inline guides/USE_JBDS.md:28]:
```suggestion
See [Red Hat Red Hat CodeReady Studio](https://developers.redhat.com/products/codeready-studio/overview/ "Get Started") on the Red Hat Developer Program website for additional information.
```

File: guides/USE_JBDS.md [hunk 3]
```diff
@@ -89,11 +89,11 @@ You can run the Arquillian tests that are included with many of the JBoss EAP 7.
...
+You may see one or more of the following errors or warnings when you import quickstart projects into Red Hat CodeReady Studio.
 
 * **JRE System Library Problem: Build path specifies execution environment JavaSE-1.6. There are no JREs installed in the workspace that are strictly compatible with this environment.**
 
-    The JBoss EAP 6 quickstarts demonstrate Java EE 6 features and are built on Java SE 6. JBoss Developer Studio requires Java 8 to run, but is still capable of launching runtimes with various versions of Java. You can ignore this warning, or you can install a JDK 6 on your machine and add it to the Eclipse Java Runtime Environment by choosing `Preferences` --> `Java` --> `Installed JREs`.
+    The JBoss EAP 6 quickstarts demonstrate Java EE 6 features and are built on Java SE 6. Red Hat CodeReady Studio requires Java 8 to run, but is still capable of launching runtimes with various versions of Java. You can ignore this warning, or you can install a JDK 6 on your machine and add it to the Eclipse Java Runtime Environment by choosing `Preferences` --> `Java` --> `Installed JREs`.
```
Comment (ncbaratta) [inline guides/USE_JBDS.md:96]: Should this be EAP 7 instead of 6?

Comment (emmartins) [reply guides/USE_JBDS.md:96]: No, from what I see that's a particular issue that applies only to EAP 6

File: guides/USE_JBDS.md (hunk 2 above)
Comment (emmartins) [reply guides/USE_JBDS.md]: Fixed

File: guides/USE_JBDS.adoc (hunk 1 above)
Comment (emmartins) [reply guides/USE_JBDS.adoc]: Fixed

**Pull request merged**